components/camera.js

```diff
@@ -113,8 +116,20 @@ export default class CameraComponent extends React.Component {
       return <Text>No access to camera</Text>;
     } else {
       return (
-          <View style={{ flex: 1 }}>
+          <View style={{ flex: 1, backgroundColor: 'purple' }}>
             <Camera style={{ flex: 1 }} type={this.state.cameraType}  ref={ref => {this.camera = ref}}>
+            {
+              this.exitCamera === undefined ? (<View/>) :(
```

Comment:
Interesting you are checking for undefined as opposed to false. I might just be misunderstanding, but I figured exitCamera was a state toggle with either true or false?